ui/ui-frontend/projects/archive-search/src/app/archive/archive-search/simple-criteria-search/simple-criteria-search.component.ts

```diff
@@ -181,130 +181,141 @@ export class SimpleCriteriaSearchComponent implements OnInit {
   }
 
   isEmpty(formData: any): boolean {
```

Comment:
Je pense qu'il faut nommer is not empty non?

Reply:
yes allé